src/app/components/work-item-quick-add/work-item-quick-add.component.less

```diff
@@ -64,6 +64,160 @@
       }
     }
   }
+
+  &__witable {
+    position: relative;
+    padding: 20px 0;
```

Comment:
use padding-top. padding-bottom separately if padding-left, padding-right:0 is not required